### PR TITLE
[DOCS-192] Webhooks UI

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Cobalt Integrations  | https://deploy-preview-171--cobalt-docs.netlify.app/integrations/ | New book |
| Get Pentest Updates with Webhooks  | https://deploy-preview-171--cobalt-docs.netlify.app/integrations/webhooks/ | Webhooks UI guide |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

### CODE REVIEW

File: content/en/Integrations/_index.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/_index.md:25]: I may need to remove images. The first one doesn't display properly.

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:13]: I feel like this is the only time we need to mention API (possible exception, whatever CX-I comes up with)

From a UI User PoV, I'm not sure they care that our Webhooks are based on our API.

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:18]: Based on my previous comment, I might:

```suggestion
When you work with our webhooks, you can monitor progress with your pentests to:

- Use polling to retrieve new information on a regular basis.
- Configure a webhook that automatically sends updates to the specified URL.
```

I don't think we need to specify "API polling", as it's functionally no different from "regular" [polling](https://en.wikipedia.org/wiki/Polling_(computer_science))

I think "updates" and "for new data" are somewhat redundant.

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:38]: I _think_ this is the authentication token from the Native or Partner integrations? Maybe we can be more specific?

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:39]: I wonder if we can/should say something like "If possible, test your connections"

Functionally, I'm concerned about firewalls which filter certain kinds of data (or stops data that's received frequently)

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:56]: Should we include a "Sign in to Cobalt" step?

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:86]: I'm "used to" the [Google Developer Style Guide](https://developers.google.com/style/tables#introductory-sentences-for-tables) practice to intro a table with a colon -- but I realize Microsoft's guide is different in this area.

```suggestion
Find solutions for common troubleshooting problems in the table below:
```

File: content/en/Integrations/_index.md
Comment (mjang-cobalt) [reply content/en/Integrations/_index.md:25]: If so, bummer. I'm glad that you're trying.

**Review by mjang-cobalt — COMMENTED** [review]
@ana-dashuk-cobalt Overall this is great. Your writing is clear, and in a couple of minutes, I understand your message. 

The rest are "details". I've made comments on what I see. I recognize that the design / details could change.

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [reply content/en/Integrations/webhooks.md:18]: Alternative:

```suggestion
When you work with an API, you can become aware of new data in the following ways:

- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information.
- Configure a webhook that automatically sends updates for new data to the specified URL.
```

File: content/en/Integrations/_index.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/_index.md:17]:
```suggestion
In the Cobalt app, navigate to the **Integrations** page to get started.
```
@mjang 
My assumption:
In Product Docs, we describe the user experience in the Cobalt app. We only have one app. We assume that the user has already signed in and is exploring the app features. That's why I'm only adding "in the Cobalt app." Maybe it's my habit from my previous projects. 
For API use cases, of course we include the steps for working with the API.

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/webhooks.md:56]:
```suggestion
1. In the Cobalt app, navigate to the **Integrations** page, and then select **Webhooks** under **Native Integrations**.
```

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:55]: @mjang-cobalt Please see my previous comment and let me know if that works.

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:13]: @mjang-cobalt I think our integrations are aimed at advanced users (in many cases, developers). So I think it's OK to keep this info,

File: content/en/Integrations/_index.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/_index.md:17]:
```suggestion
In the Cobalt app, navigate to the **Integrations** page to get started.
```

I'm assuming that the user has already signed in and is now exploring the app functionality. We only have one app, so I think it's OK to mention "in the Cobalt app." Please let me know if it works. Maybe it's my habit from my previous projects.

File: content/en/Integrations/_index.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/_index.md:19]: @mjang-cobalt I would avoid adding much info about these integration types. We discussed it earlier. It looks like we're productizing different types of integrations, which is not our focus.

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/webhooks.md:39]:
```suggestion
1. Make sure that the **URL** where you want to receive notifications is valid and your services work properly. If possible, test your connections.
```

File: content/en/Integrations/_index.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/_index.md]: @mjang-cobalt I moved the screenshots to the `integrations` folder.

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md]: @mjang-cobalt It's an authentication token from the user's app. We use it to authorize POST requests to their webhook URL.

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/webhooks.md:74]:
```suggestion
- To make a webhook **inactive**, use the toggle under **Active**.
```

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/webhooks.md:77]:
```suggestion
  - You can make a webhook inactive without deleting it.
```

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [reply content/en/Integrations/webhooks.md:56]: You do now say "In the Cobalt app"... 
(or did I miss that earlier?)

Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:56]: I made a commit to the file, I should have added a suggestion.

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:92]: One more item. (My bad for not catching this earlier.)

In two other locations ([here](https://github.com/cobalthq/cobalt-product-public-docs/blame/main/content/en/Getting%20started/Assets/asset-description.md#L110) and [here](https://github.com/cobalthq/cobalt-product-public-docs/blame/3a56912d250538dbb872e81685f18802abbba604/content/en/Getting%20started/what-to-expect.md#L58)), we've said "Contact your Customer Success Manager (CSM) or support@cobalt.io."

Maybe it's time to set up a "partial" with that statement?

**Review by mjang-cobalt — APPROVED** [review]
Approving with one additional comment. As discussed (for the record), we won't merge this PR until the changes are pushed to the product

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:92]: @mjang-cobalt Here is a minor PR for that: https://github.com/cobalthq/cobalt-product-public-docs/pull/173

File: content/en/Integrations/webhooks.md
Comment (mjang-cobalt) [inline content/en/Integrations/webhooks.md:79]: Thinking ahead to when we implement search, I think we need a slightly more descriptive title:

```suggestion
## Troubleshoot Webhooks
```

File: content/en/Integrations/webhooks.md
Comment (inikolov89) [reply content/en/Integrations/webhooks.md:13]: Some additional info:

The webhooks use a skinny payload, containing only the public id of the entity. This is for security reasons. 

So to be able to retrieve information that actually was updated about the entity, the users will need to use e.g. our public API. 

Maybe we should add some documentation on how to retrieve information about a webhook event with the API?
cc: @eugenerevzin

**Review by eugenerevzin — APPROVED** [review]

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:13]: @inikolov89 I think adding more info about APIs is out of scope for this task since we're only describing the Webhooks UI in this guide.
I plan to add API use cases to our Product Documentation with more details about using webhook APIs. You can see an example use case for assets here: https://developer.cobalt.io/apiusecases/create_asset/

File: content/en/Integrations/webhooks.md
Comment (ana-dashuk-cobalt) [inline content/en/Integrations/webhooks.md:54]: @inikolov89 As a workaround, I added this paragraph to explain how users can get more details about webhook events.

Comment (ana-dashuk-cobalt) [reply content/en/Integrations/webhooks.md:54]: @inikolov89 Since we're getting closer to the release, can you please approve the PR with the suggested changes? Thank you.

**Review by inikolov89 — APPROVED** [review]
Great work!

**Pull request merged**
